py3status/modules/wwan.py

```diff
@@ -0,0 +1,565 @@
+# -*- coding: utf-8 -*-
+"""
+Display WWANs, IP addresses, signals, properties, and more.
+
+Configuration parameters:
+    cache_timeout: refresh interval for this module (default 5)
+    format: display format for this module
+        *(default '\?color=state WW: [\?if=state_name=connected '
+        '({signal_quality_0}% at {m3gpp_operator_name}) '
+        '[{format_ipv4}[\?soft  ]{format_ipv6}]|{state_name}]'
+        '[ SMS {messages} [{format_message}]]')*
+    format_ipv4: display format for ipv4 network (default '[{address}]')
+    format_ipv6: display format for ipv6 network (default '[{address}]')
+    format_message: display format for SMS messages
+        (default '\?if=index<2 {number} [\?max_length=10 {text}...]')
+    format_message_separator: show separator if more than one (default ' ')
+    format_notification: specify notification to use (default None)
+    format_stats: display format for statistics (default '{duration_hms}')
+    modem: specify a modem device to use. otherwise, auto (default None)
+    thresholds: specify color thresholds to use
+        (default [(0, 'bad'), (11, 'good')])
+
+Format placeholders:
+    {access_technologies}           network speed, in bit, eg 192
+    {access_technologies_name}      network speed names, eg LTE
+    {current_bands}                 modem band, eg 1
+    {current_bands_name}            modem band name, eg GSM/GPRS/EDGE 900 MHz
+    {format_ipv4}                   format for ipv4 network config
+    {format_ipv6}                   format for ipv6 network config
+    {format_message}                format for SMS messages
+    {format_stats}                  format for network connection statistics
+    {interface_name}                network interface name, eg wwp0s20f0u2i12
+    {m3gpp_registration_state_name} network registration state name, eg HOME
+    {m3gpp_registration_state}      network registration state, eg 1
+    {m3gpp_operator_code}           network operator code, eg 496
+    {m3gpp_operator_name}           network operator name, eg Py3status Telecom
+    {message}                       number of messages, eg 2
+    {messages}                      total number of messages, eg 30
+    {signal_quality_0}              signal quality percentage, eg 88
+    {signal_quality_1}              signal quality refreshed, eg True/False
+    {state}                         network state, eg 0, 7, 11
+    {state_name}                    network state name, eg searching, connected
+
+format_ipv4 placeholders:
+    {address} ip address
+    {dns1}    dns1
+    {dns2}    dns2
+    {gateway} gateway
+    {mtu}     mtu
+    {prefix}  netmask prefix
+
+format_ipv6 placeholders:
+    {address} ip address
+    {dns1}    dns1
+    {dns2}    dns2
+    {gateway} gateway
+    {mtu}     mtu
+    {prefix}  netmask prefix
+
+format_message placeholders:
+    {index}   message index
+    {message} message received, eg: '+33601020304: hello how are you?'
```

Comment:
Is this `{message}` placeholder even available? I don't see it in the code. Maybe you ought to test all examples and placeholders tomorrow. :+1:

Reply:
Confirm you that {message} exists ;)

Reply:
@cyrinux In `format_message`? Not `format`? If yes, how?

Reply:
Yet thinking on this...and try re-QA some format